plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -107,8 +107,11 @@ contract('PaymentExitGame - End to End Tests', ([_, richFather, bob]) => {
         this.erc20Vault = await Erc20Vault.new(this.framework.address);
         await this.erc20Vault.setDepositVerifier(erc20DepositVerifier.address);
 
-        await this.framework.registerVault(1, this.ethVault.address);
-        await this.framework.registerVault(2, this.erc20Vault.address);
+        await this.framework.registerVault(ETH_VAULT_ID, this.ethVault.address);
+        await this.framework.registerVault(ERC20_VAULT_ID, this.erc20Vault.address);
+
+        await this.framework.addExitQueue(ETH_VAULT_ID, ETH);
+        await this.framework.addExitQueue(ERC20_VAULT_ID, this.erc20Vault.address);
```

Comment:
There is a test of `Given ERC20 token added to the PlasmaFramework` and would add the queue for ERC20 token there. So it shouldn't be done here. However, just realize we can add the queue twice seems like a **bug**!

(wait....I see the code should actually block this, how come this works.........

Reply:
Thankfully there is no bug and we have it covered by a unit test.
I messed up a bit, those are not the same calls, the first one should be removed:
`
await this.framework.addExitQueue(ERC20_VAULT_ID, this.erc20Vault.address);
await this.framework.addExitQueue(ERC20_VAULT_ID, this.erc20.address);
`

Reply:
ahhh......`erc20Vault` vs `erc20`